### PR TITLE
Adding UI elements

### PR DESCRIPTION
**Pull request by paulSanchezMarmelab**
HUD_zoomed_3

https://trello.com/c/a5AXazM9/58-1-as-hayao-i-want-to-know-whose-its-the-turn

- [X] using partials
- [X] Current player
- [X] Game state displayed (ongoing / won / lost / draw)
- [X] Number of remaining tiles

### CODE REVIEW

**Review by Nitix — CHANGES_REQUESTED** [review]

File: webapp/public/hud.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
+:root{
+    --light: white;
+    --dark: black;
+    --gray: grey;
+    --gold: rgba(209, 209, 0, 0.767);
+    --xs: 1.5rem;
```
Comment (adguernier) [inline webapp/public/hud.css:6]:
```suggestion
    --tile-whidth-xs: 1.5rem;
```

File: webapp/public/hud.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  background-color: var(--gray);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: 900;
+  font-family: "Font Awesome 6 Free";
+  z-index: -1;
+}
+
+.turn-counter {
```
Comment (adguernier) [inline webapp/public/hud.css:69]:
```suggestion
.turn-counter, hexagon-container {
```

File: webapp/views/partials/HUD.hbs [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<aside class="playersHUD">
```
Comment (adguernier) [inline webapp/views/partials/HUD.hbs:1]:
```suggestion
<aside class="players-hud">
```

File: webapp/views/index.hbs [hunk 4]
```diff
@@ -5,11 +5,15 @@
     <title>App</title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="app.css">
+    <link rel="stylesheet" href="hud.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
```
[the remark this answers is not in the document]
Comment (paulSanchezMarmelab) [reply webapp/views/index.hbs]: I removed material icons and switched to font-awesome.

**Pull request merged**